### ISSUE TITLE
Campana no funciona

### ISSUE THREAD

**Issue by peletubi** [Daher-Maluf/task-app#10]
No se si esto era una de las cosas que habia pendientes, creo que si, pero por si acaso lo dejo aqui

Hay tareas no leidas y la campana tiene el numero 0 y no muestra nada cuando le pinchas

![Image](https://github.com/Daher-Maluf/task-app/assets/633665/789d8c0b-90dd-41c1-81d1-f426217e6d9e)

**Comment by Daher-Maluf** [comment]
@peletubi Sí, debería responder en tiempo real a los cambios. Sin embargo, solo muestra las tareas no leídas del técnico que ha iniciado sesión.
